### DISCUSSION TITLE
Can I use it with Podman? I am facing 503 Temporarily Not Available error. I suspect it is due to the docker-gen.

### DISCUSSION THREAD

**Discussion by stephenllh** [nginx-proxy/nginx-proxy#2258] [Q&A]

**Comment by heyakyra** [comment]
What version? Was [working well](https://github.com/nginx-proxy/nginx-proxy/discussions/2112#discussioncomment-16929922) for me with `podman` until the latest releases https://github.com/nginx-proxy/nginx-proxy/issues/2762
